src/frontend/src/btc/components/fee/UtxosFeeContext.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+	import { debounce, isNullish, nonNullish } from '@dfinity/utils';
+	import { getContext } from 'svelte';
+	import { selectUtxosFee as selectUtxosFeeApi } from '$btc/services/btc-send.services';
+	import { UTXOS_FEE_CONTEXT_KEY, type UtxosFeeContext } from '$btc/stores/utxos-fee.store';
+	import { authIdentity } from '$lib/derived/auth.derived';
+	import { nullishSignOut } from '$lib/services/auth.services';
+	import type { NetworkId } from '$lib/types/network';
+	import { isNetworkIdBitcoin, mapNetworkIdToBitcoinNetwork } from '$lib/utils/network.utils';
+
+	export let amount: number | undefined = undefined;
+	export let networkId: NetworkId | undefined = undefined;
+
+	const { store } = getContext<UtxosFeeContext>(UTXOS_FEE_CONTEXT_KEY);
+
+	const loadEstimatedFee = async () => {
+		if (isNullish($authIdentity)) {
```

Comment:
Should we have a test for this edge case?

Reply:
Added.